completion_aggregator/tasks.py

```diff
@@ -67,16 +76,20 @@ def _update_aggregators(user, course_key, block_keys=frozenset(), force=False):
             The user whose aggregators need updating.
         course_key (opaque_keys.edx.keys.CourseKey):
             The course in which the aggregators need updating.
-        block_key (list[opaque_keys.edx.keys.UsageKey]):
+        block_keys (list[opaque_keys.edx.keys.UsageKey]):
              A list of completable blocks that have changed.
         force (bool):
             If True, update aggregators even if they are up-to-date.
 
     """
     from xmodule.modulestore.django import modulestore   # pylint: disable=import-error
-    log.debug("Updating aggregators in %s for %s", course_key, user)
-    updater = AggregationUpdater(user, course_key, modulestore())
-    updater.update(block_keys, force)
+    log.info("\n\n\nUpdating aggregators in %s for %s. Changed blocks: %s\n\n\n", course_key, user, block_keys)
+    try:
+        updater = AggregationUpdater(user, course_key, modulestore())
+        updater.update(block_keys, force)
+    except Exception as e:
```

Comment:
Pylint doesn't complain about it because the exception gets reraised after logging.  But yeah, I think this is unnecessary.  Raising an exception should provide sufficient logging.